recipes/cufflinks/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cufflinks" %}
```

Comment:
Can we rename this recipe, to python-cufflinks to avoid naming confusions with the bioconda package?

Reply:
oooof gross.

Reply:
There are other examples for this in conda-forge, e.g. python-igraph, in order to distinguish it from the igraph package.

Reply:
I think that makes sense in places where the python wrapper has the same name I.e. kafka/kafka-python. I just think the python cufflinks is the more established package, and the other package should change names, but I suppose thats first-mover privilege.

Reply:
if it generates any noise or confusion in the cufflinks package, we'll just refer them over to you guys anyway, so its no inconvenience on our end.

Reply:
Yeah, in order to ensure reproducibility (in the scientific sense), changing the name of the other cufflinks package is not an option.

Reply:
> we'll just refer them over to you guys anyway, so its no inconvenience on our end.

Please rethink your attitude here, Bioconda and conda-forge are working together not against each other. Thanks.

Reply:
we are neither though. I'm saying if people ask "why doesn't `conda install cufflinks`" work, we will refer them to this PR.

Reply:
or to rephrase, "we don't have an issue with this if you don't"

Reply:
> or to rephrase, "we don't have an issue with this if you don't"

That sounds way cooler :)
Imho, and this has historical reasons, we missed to prefix packages or invent some other namespace systems. For example, R packages we also prefix `r-` and similar for `perl-`. Now we need to deal with the consequences. Thanks for your understanding!